agent/src/main/java/com/microsoft/applicationinsights/agent/internal/agent/AdvancedAdviceAdapter.java

```diff
@@ -153,7 +153,7 @@ protected String getMethodName() {
         return methodName;
     }
 
-    protected Type getReturnType() {
+    public Type getReturnType() {
```

Comment:
Why did the visibility change here?

Reply:
![image](https://user-images.githubusercontent.com/21194782/56831671-973d3c80-6861-11e9-8005-c52810757819.png)

Reply:
`@Override`